src/module/apps/itemImport/helper/ImportHelper.ts

```diff
@@ -186,7 +190,7 @@ export class ImportHelper {
      * @param typeKey The item type to translate. Tends to be plural.
      * @param listKey The item to translate. Tends to be singular.
      */
-    public static ExtractItemTranslation(jsonItemsi18n, typeKey, listKey) {
+    public static ExtractItemTranslation(jsonItemsi18n, typeKey, listKey) : {} {
```

Comment:
Why the `:{}` return type? 
That would mean about 'any non-nullish type'. It can also be just removed to allow indirect typing to take place and no type errors occure.